tests/eventbus_test.py

```diff
@@ -0,0 +1,208 @@
+import os
+import tempfile
+
+import mock
+from testify import assert_equal
+from testify import setup
+from testify import teardown
+from testify import TestCase
+
+from tron import eventbus
+
+
+class EventBusTestCase(TestCase):
+    @setup
+    def setup(self):
+        self.log_dir = tempfile.TemporaryDirectory(prefix="tron_eventbus_test")
+        self.eventbus = eventbus.make_eventbus(self.log_dir.name)
+
+    @teardown
+    def teardown(self):
+        self.log_dir.cleanup()
+
+    @mock.patch('tron.eventbus.reactor', autospec=None)
+    def test_start(self, reactor):
+        self.eventbus.setup_log_dir = mock.Mock()
+        self.eventbus.sync_load_log = mock.Mock()
+        reactor.callLater = mock.Mock()
+        self.eventbus.start()
+        assert self.eventbus.setup_log_dir.call_count is 1
+        assert self.eventbus.sync_load_log.call_count is 1
+        assert reactor.callLater.call_count is 1
+
+    @mock.patch('tron.eventbus.log', autospec=None)
+    def test_setup_log_dir(self, _):
+        os.rmdir(self.log_dir.name)
+        self.eventbus.event_log = {"foo": "bar"}
+        self.eventbus.setup_log_dir()
+        assert os.path.exists(self.log_dir.name)
+        assert os.path.exists(os.path.join(self.log_dir.name, "current"))
+
+        new_eb = eventbus.EventBus(self.log_dir.name)
+        new_eb.sync_load_log()
+        assert new_eb.event_log == self.eventbus.event_log
+
+    def test_shutdown(self):
+        assert not self.eventbus.must_shutdown
+        self.eventbus.shutdown()
+        assert self.eventbus.must_shutdown
+
+    def test_publish(self):
+        evt = {'id': 'foo'}
+        self.eventbus.publish(evt)
+        assert self.eventbus.publish_queue.pop() is evt
+
+    def test_subscribe(self):
+        ps = ('foo', 'bar', 'cb')
```

Comment:
shouldn't the callback be a function? otherwise I think you'll have to do something [here](https://github.com/Yelp/Tron/pull/497/files#diff-bbce3bb8c085b14e354c7b20df758cbdR162) to convert a string to a function?

Reply:
yep, it's a function. this is just a test to check we're getting the right thing on event queue, we don't really care about the values.